onnxruntime/python/tools/quantization/calibrate.py

```diff
@@ -367,3 +367,27 @@ def calibrate(model_path,
 
     print("Calibrated,quantized parameters calculated and returned.")
     return quantization_params_dict
+
+def tensorrt_calibrate(model_path,
```

Comment:
this function doesn't necessarily seem tensorrt specific.
and it mostly seems the same as calibrate().
could calibrate() have been modified to support the variations that we need?
 i.e. skip calculating quantization parameters.